modules/op-release-notes-1-17.adoc

```diff
@@ -0,0 +1,187 @@
+
+[id="new-features-1-17_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.17:
+
+[id="pipelines-new-features-1-17_{context}"]
+=== Pipelines
+
+* With this release, you can configure multiple Git providers by using the Git resolver, or you can add multiple configurations for the same Git provider, in the `TektonConfig` custom resource (CR). You can then specify a Git configuration to use in different task runs and pipeline runs by using the `configKey` parameter.
++
+.Example of multiple Git provider configurations
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  pipeline:
+    git-resolver-config:
+      # configuration 1 
+      fetch-timeout: "1m"
+      default-url: "https://github.com/tektoncd/catalog.git"
+      default-revision: "main"
+      scm-type: "github"
+      server-url: ""
+      api-token-secret-name: ""
+      api-token-secret-key: ""
+      api-token-secret-namespace: "default"
+      default-org: ""
+      # configuration 2
+      test1.fetch-timeout: "5m"
+      test1.default-url: ""
+      test1.default-revision: "stable"
+      test1.scm-type: "github"
+      test1.server-url: "api.internal-github.com"
+      test1.api-token-secret-name: "test1-secret"
+      test1.api-token-secret-key: "token"
+      test1.api-token-secret-namespace: "test1"
+      test1.default-org: "tektoncd"
+      # configuration 3
+      test2.fetch-timeout: "10m"
+      test2.default-url: ""
+      test2.default-revision: "stable"
+      test2.scm-type: "gitlab"
+      test2.server-url: "api.internal-gitlab.com"
+      test2.api-token-secret-name: "test2-secret"
+      test2.api-token-secret-key: "pat"
+      test2.api-token-secret-namespace: "test2"
+      test2.default-org: "tektoncd-infra"
+# ...
+----
++
+.Example of using a `test1` configuration in a pipeline run
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: example
+spec:
+  pipelineRef:
+    resolver: git
+    params:
+    # ...
+    - name: configKey
+      value: test1
+# ...
+----
+
+* With this update, you can now monitor the running `PipelineRun` resources at the namespace, pipeline, and pipeline run level, in addition to the cluster level. You can configure the level through the `TektonConfig` custom resource by setting the `metrics.running-pipelinerun.level` parameter value to `namespace`, `pipeline`, or `pipelinerun`. The default value is empty, which sets the monitoring at cluster level.
++
+.Example of monitoring the running pipeline run at the namespace level
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    # ...
+    metrics.running-pipelinerun.level: namespace
+# ...
+----
+
+[id="cli-new-features-1-17_{context}"]
+=== CLI
```

Comment:
This has nothing to do with CLI, it's about ecosystem tasks, maybe section "Operator"?